esp/esp/program/modules/base.py

```diff
@@ -292,6 +292,11 @@ def useTemplate(self):
         """ Use a template if the `mainView' function doesn't exist. """
         return (not self.main_view)
 
+    def isAdminPortalFeatured(self):
+        """Don't display in the long list of additional modules if it's already featured
+        in the main portion of the admin portal"""
+        return self.module.admin_title in ["Program Dashboard (AdminCore)", "Course Materials", "Program Vitals (part of Dashboard)", "Teacher Availability Checker", "Volunteer Management", "Manage Classes (part of Dashboard)", "AJAX Scheduling (AJAXSchedulingModule)", "Class Search", "Resource Management", "Communications Panel for Admin", "User Morphing Capability", "Nametag Generation", "User List Generator"]
```

Comment:
"Teacher Availability Checker" and "Class Search" are not featured on the admin portal

Reply:
If the reason to not include the "Class Search" module is because it is a duplicate of the "Class Flags" module, I would argue you should switch them.

Reply:
Nope, just a mistake. Thanks for the catch.

Reply:
Also, can we exclude the Class Flags module, since it just redirects to the Class Search module?

Reply:
fixed in commit 78a3f3f

Reply:
Uhh...the Class Flags module is not listed in that list

Reply:
added in cdeabbf